### DISCUSSION TITLE
ΓΕ1 Μέρος Γ βίντεο παιχνίδι από την αρχή

### DISCUSSION THREAD

**Discussion by ghost** [sdy60/syllabus#33] [Q&A] [answered]
Κύριε @epidrome προετοιμάζω την απάντηση στην εργασία και συνειδητοποίησα ότι στο μέρος Γ λέτε να κάνουμε **μικρές επεκτάσεις**. Έχω φτιάξει ένα παιχνίδι από την αρχή. Να σας το παραδώσω αυτό ή να το πετάξω;

**Comment by epidrome** [comment, marked as answer]
Το μέρος Γ είναι πολύ εύκολο για κάποιον που ασχολείται ενεργά με τον προγραμματισμό και προσβάσιμο για κάποιον που έχει καιρό να ασχοληθεί. Επομένως θα εκτιμηθεί πολύ θετικά αν κάποιοι προσαρμόσουν τον βαθμό δυσκολίας στις ικανότητες τους, π.χ., φτιάχνοντας κάτι από την αρχή, μπράβο σας!
